resources/keycloak/realm.json

```diff
@@ -5150,28 +5150,33 @@
         ]
       }
     },
-    "groups": [
+    "federatedUsers": [
       {
-        "id": "1d271cc0-96b8-434d-9c87-375cd789897a",
-        "name": "ADMIN",
-        "path": "/ADMIN",
-        "attributes": {},
-        "realmRoles": ["ADMIN"],
-        "clientRoles": {
-          "realm-management": ["manage-users", "realm-admin"]
+        "id": "f:a6d0ac0a-279e-40f6-a226-81cc7d5be501:e3f4884b-7cb4-46cc-a353-1314ac2ff03e",
```

Comment:
These are two entries which add the keycloak-side of the user's data, since keycloak no longer stores the base user information. The keycloak-side data includes the email_verified attribute and similar, as well as the user's password (credential block). Then, the rest of the user's base info (user_id, keycloak_id, name, ...) is all in postgres

Reply:
So this is just relating to the email_verified attribute and credential information at a high level and not a particular user? The way this read with being under "federatedUsers" it makes it look like we're adding in two explicit users.

Reply:
I don't really understand why they are different entries, but they both apply to the same user (you can see the first half of the id is the same). The first half of the id corresponds to the user id injected into postgres, which is how they are applied